server/auto_merge.go

```diff
@@ -6,7 +6,6 @@ package server
 import (
 	"context"
 	"fmt"
-
```

Comment:
It's idiomatic for go code to group all imports from the `std` lib together. This is also the style that https://pkg.go.dev/golang.org/x/tools/cmd/goimports?tab=doc enforces.

Why drop this style?

Reply:
I didn't know about this tool. Makes sense to add it to the CI. I added it.

Reply:
I don't have any objections on using `goimports`. Other Mattermost project using `golangci-lint` as a metalinter. Hence, it makes sense to also integrate `golangci-lint` into this repo, which can also run `goimports`.